sklearn/preprocessing/_encoders.py

```diff
@@ -171,6 +171,8 @@ class OneHotEncoder(_BaseEncoder):
 
     Read more in the :ref:`User Guide <preprocessing_categorical_features>`.
 
+    .. versionchanged:: 0.20
```

Comment:
I don't think we should have `versionchanged` without specifying what's changed. @rth, is this a convention we have adopted?

Reply:
I agree, must have missed it.